src/app/modules/project/details/details.component.html

```diff
@@ -85,14 +85,15 @@ <h1 class="project-title">{{ project.name }}</h1>
           see more tags&hellip;
         </a>
       </div>
+
       <p class="project-description large">
-        {{ project.description }}
+        <quill-view class="dex-quill-viewer" [content]="project.description" format="text"></quill-view>

```

Comment:
Just a question: Did you check if this makes XSS possible?
Quill has an option to sanitise HTML, but this is disabled by default. You are using the "text" format so I don't know if this is an issue.

Reply:
The following sanitizations take place:
- Quill itself performs sanitization on input
- Description content is also put through a sanitizer in the backend
- Angular escapes all content retrieved through the HttpClient
- Angular performs last sanitization before setting the content in the innerHTML